payments/notifications/types.py

```diff
@@ -27,3 +27,4 @@ class NotificationType(TextChoices):
         "additional_product_order_approved",
         _("Additional product order approved"),
     )
+    ORDER_CANCELLED = ("order_cancelled", _("Confirmation"))
```

Comment:
Is the message consistent with other NotificationType fields? Other fields (NEW_BERTH_ORDER_APPROVED etc) have a more specific message.

Reply:
According to the translations we got from Kalle, that should be the email subject
https://helsinkicity.slack.com/archives/CBA1QHTHN/p1610962478024200?thread_ts=1610961964.020500&cid=CBA1QHTHN